cicd/pipelines/remote-pipelines-tasks-resolvers.adoc

```diff
@@ -0,0 +1,52 @@
+:_content-type: ASSEMBLY
+[id="remote-pipelines-tasks-resolvers"]
+= Specifying remote pipelines and tasks using resolvers
+include::_attributes/common-attributes.adoc[]
+:context: remote-pipelines-tasks-resolvers
+
+toc::[]
+
+By design, pipelines and tasks are reusable blocks for your CI/CD processes. You can reuse pipelines or tasks that you previously developed, without having to copy and paste their definitions. Moreover, you can reuse pipelines or tasks developed by other teams. These pipelines or tasks can be available from several types of sources, from other namespaces on your cluster to public catalogs.
+
+In a pipeline run resource, you can specify a pipeline from an existing source. In a pipeline resource or a task run resource, you can specify a task from an existing source.
+
+In these cases, the _resolvers_ in {pipelines-title} retrieve the pipeline or task definition from the specified source at run time.
+
+The following resolvers are available in a default installaton of {pipelines-title}:
+
+* _Hub resolver_: retrieves a task or pipeline from the Pipelines Catalog available on {artifact-hub} or {tekton-hub}.
```

Comment:
Not critical, but this might look better using the definition formatting:

```
Hub resolver:: retrieves a task or pipeline from the Pipelines Catalog available on {artifact-hub} or {tekton-hub}.
...
```